### PR TITLE
Backend: Facebook webhook action and copy_proto update

### PR DESCRIPTION
**Pull request by EdenComp**
Also fixes ARE-320
Also fixes ARE-366

### CODE REVIEW

**Review by RezaRahemtola — APPROVED** [review]

File: backend/back/src/webhooks/services/facebook-webhook.service.ts [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+import { Type } from "class-transformer";
+import { createHmac, timingSafeEqual } from "crypto";
+import { GrpcService } from "../../grpc/grpc.service";
+
+export class FacebookChallengeHub {
+	@IsString()
+	"hub.mode": string;
+	@IsString()
+	"hub.verify_token": string;
+	@Type(() => Number)
```
Comment (Croos3r) [inline backend/back/src/webhooks/services/facebook-webhook.service.ts:19]: This is not needed as the conversion is enabled in the global validation pipe

File: backend/back/src/webhooks/services/facebook-webhook.service.ts [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+	verb: string;
+	created_time: number;
+	message: string;
+	from: {
+		id: string;
+		name: string;
+	};
+};
+
+type FacebookWebhookBody = FacebookBaseBody<FacebookFeedStatusEntry>; // eslint-disable-line
```
Comment (Croos3r) [inline backend/back/src/webhooks/services/facebook-webhook.service.ts:51]: why ?

File: backend/supervisor/jobs/images.go [hunk 3]
```diff
@@ -1,6 +1,7 @@
 package jobs
 
 var JobToImage = map[string]string{
+	"facebook-on-status-create":              "",
```
Comment (Croos3r) [inline backend/supervisor/jobs/images.go:4]: You left it blank because it doesnt have to be started as a job ?

Comment (EdenComp) [reply backend/supervisor/jobs/images.go:4]: Exactly, I don't want to create a new map in the back to determine whether we should start a job or not so I added it to the images map on the supervisor. If the value is "" it means that the AREA is valid but it doesn't need a job

**Review by Croos3r — APPROVED** [review]

**Pull request merged**